src/redux/mapStoreToProps.ts

```diff
@@ -1,23 +1,24 @@
 export interface Song {
-  title: string;
```

Comment:
@Unesdala this is where you can import the `Song` type to use in the other places